src/AzureAuth/Commands/Ado/CommandToken.cs

```diff
@@ -53,7 +53,7 @@ public enum OutputMode
         private IEnumerable<AuthMode> AuthModes { get; set; } = new[] { AuthMode.Default };
 
         [Option(CommandAad.DomainOption, Description = CommandAad.DomainHelpText)]
-        private string Domain { get; set; }
+        private string Domain { get; set; } = AzureAuth.Ado.Constants.PreferredDomain;

```

Comment:
Can we change the Domain help text to match the CommandAdoPat help text? That help indicates the default on the ADO command is Microsoft, while the generic Aad command does not (and has no default)